numba/dppl/compiler.py

```diff
@@ -118,7 +119,8 @@ def compile_kernel(device, pyfunc, args, access_types, debug=False):
     cres = compile_with_dppl(pyfunc, None, args, debug=debug)
     func = cres.library.get_function(cres.fndesc.llvm_func_name)
     kernel = cres.target_context.prepare_ocl_kernel(func, cres.signature.args)
-    oclkern = DPPLKernel(device_env=device,
+    oclkern = DPPLKernel(context=cres.target_context,
```

Comment:
A comment explaining why the "context" param was added will be good.

Reply:
Comment added.